src/core/error.extern.js

```diff
@@ -21,3 +21,7 @@
 
 /** @type {undefined|Array<*>} */
 Error.prototype.messageArray;
+
+// Error prototype extended by AMP
+/** @type {undefined|boolean} */
+Error.prototype.expected;
```

Comment:
This property is AMP-only and not part of core, please remove from here. If desired it can live in `src/error-reporting.extern.js` since it's consumed only by error reporting code

Reply:
Right...you already said that...sorry! I'll put in `amp.extern.js` for now